tool/cli.ts

```diff
@@ -597,9 +599,9 @@ program
       if (CONTENT_TRANSLATED_ROOT) {
         roots.push(CONTENT_TRANSLATED_ROOT);
       }
-      const map = gatherGitHistory(
+      const map = await gatherGitHistory(
         roots,
-        loadHistory && fs.existsSync(loadHistory) ? loadHistory : null
+        loadHistory && (await fse.pathExists(loadHistory)) ? loadHistory : null
```

Comment:
Just delete `loadHistory`, or set to null, above if `fs.access(loadHistory)` fails